nucypher/characters/chaotic.py

```diff
@@ -133,15 +134,16 @@ class Felix(Character, NucypherTokenActor):
     TEMPLATE_NAME = 'felix.html'
 
     # Intervals
-    DISTRIBUTION_INTERVAL = 60*60   # seconds (60*60=1Hr)
-    DISBURSEMENT_INTERVAL = 24      # (24) hours
-    STAGING_DELAY = 10              # seconds
+    DISTRIBUTION_INTERVAL = 60  # seconds
+    DISBURSEMENT_INTERVAL = 24 * 365  # only distribute tokens to the same address once each YEAR.
+    STAGING_DELAY = 10        # seconds
 
     # Disbursement
     BATCH_SIZE = 10                 # transactions
-    MULTIPLIER = 0.95               # 5% reduction of previous stake is 0.95, for example
-    MINIMUM_DISBURSEMENT = 1e18     # NuNits
-    ETHER_AIRDROP_AMOUNT = int(2e18)  # Wei
+    MULTIPLIER = Decimal('0.9')     # 10% reduction of previous stake is 0.9
```

Comment:
```suggestion
    MULTIPLIER = Decimal('0.9')     # 10% reduction of previous disbursement is 0.9
```

Reply:
If we're merging into master should we just remove this since it's no longer being used?